blueprints/app/files/config/ember-try.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  scenarios: [
+    {
+      name: 'ember-1.10',
```

Comment:
Including this scenario feel like ember-cli would have the opinion of encouraging 1.10 compatibility. I'd like some specific feedback on whether people feel this is appropriate